src/templates/header/partials/logo.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="c-logo">
     <a href="/" aria-label="Just Eat Homepage">
         <div class="c-logo-img">
-            <img alt="{{ i18n "logoAltText" }}" src="{{ logoUrl }}" width="152" height="30" />
+            <img alt="{{ i18n "logoAltText" }}" src="{{ logoUrl }}" />
```

Comment:
doesn't the height and width need to be specified in the sass file at least?

Reply:
I added width and height to c-logo-img class and image takes space of the container.

Reply:
weird, unless we're specifying width to be 100% somewhere else then it will overflow if it's not the correct width right

Reply:
We use svgs here, so as I understand, if aspect ratio is the same and there is no width and height in svg file then svg will fit parent element. But I can add width and height for img... It is just even with width and height if view box of svg is different, image will look broken. Feel free to correct me if I am wrong

Reply:
ah makes sense if it's svg 👍